www/src/components/CodeEditor/CodeEditor.module.scss

```diff
@@ -75,8 +95,33 @@
   background-color: $c-divider;
 }
 
+.PreviewHeader {
+  @include f-note;
+  padding-top: 10px;
+  padding-bottom: 5px;
```

Comment:
I'm going to merge this PR for now, but I would like to chat about this with you (at your convenience)